### ISSUE TITLE
No access to Options Dialog

### ISSUE THREAD

**Issue by mgbarton** [opto/Expression-Search-NG#35]
Thanks for your efforts!

I'm running TB 91.8.0.

Based on reading the instructions there should be an Options Dialog where one can do things like turn off tool tips.  However there's no wrench Icon for this tool in the 'Manage Your Extensions' screen, and I don't see any other way to activate it.

**Comment by opto** [comment]
correct. Currently, that is in the status bar, there is a blue search glass where you can access the options, please click the searchglass
